config/locales/en/errors.yml

```diff
@@ -151,21 +151,21 @@ en:
         steps/case/has_case_concluded_form:
           attributes:
             has_case_concluded:
-              inclusion: Select yes if your client has the case concluded
+              inclusion: Select yes if the case has concluded
             date_case_concluded:
-              blank: Date case was concluded cannot be blank
-              blank_day: Date case was concluded must include a day
-              blank_month: Date case was concluded must include a month
-              blank_year: Date case was concluded must include a year
-              blank_day_month: Date case was concluded must include a day and month
-              blank_day_year: Date case was concluded must include a day and year
-              blank_month_year: Date case was concluded must include a month and year
+              blank: Enter when the case concluded
+              blank_day: Date the case concluded must include a day
+              blank_month: Date the case concluded must include a month
+              blank_year: Date the case concluded must include a year
+              blank_day_month: Date the case concluded must include a day and month
+              blank_day_year: Date the case concluded must include a day and year
+              blank_month_year: Date the case concluded must include a month and year
               invalid: Enter a valid date
```

Comment:
@ anyone looking - Ripan and I discussed and agreed that this and the following 4 errors were more specific than what is in the spreadsheet so left them alone. I am confirming with Chris O